package.json

```diff
@@ -54,7 +54,7 @@
     "build": "cargo build --workspace --target wasm32-wasi --release && cargo run",
     "build:types:preview2-shim": "PREVIEW2_SHIM_TYPES=* cargo build -p jco",
     "lint": "eslint -c eslintrc.cjs lib/**/*.js packages/*/lib/**/*.js",
-    "test": "mocha -u tdd test/test.js --timeout 120000"
+    "test": "rm -rf $(pwd)/node_modules/@bytecodealliance/jco/lib && ln -s $(pwd)/lib $(pwd)/node_modules/@bytecodealliance/jco/ && mocha -u tdd test/test.js --timeout 120000"
```

Comment:
This shouldn't be necessary, and is brittle giving the lib folder special precedence. Any code that uses jco should refer to the local `src/jco.js` path.

Reply:
it is failing without it because the adapters are incompatible with jco in npm (being pulled from componentize-js dependency)

Reply:
Ah I see, in that case, this actually seems like a great workaround, thanks for explaining!